INN/umbrella-currentorg
# LTW Editor custom user role

## Changes

This pull request makes the following changes:

- Adds a new `LTW Editor` custom role for users to be able to add, manage, update, delete LTW projects and their taxonomies

## Why



For #152

## Testing/Questions

Features that this PR affects:

- New custom user role
- Capabilities for projects and project taxonomies


Questions that need to be answered before merging:

- [x] **NOTE:** WP 5.4.2 needs to be installed for this to work correctly
- [x] Any security concerns?
- [ ] I added the new projects capabilities to administrator and editor roles as well. Do we need to add it to any other roles?
- [ ] Why is the landing page CPT showing for users with this role? Removing 
```
'manage_terms'  => 'edit_projects',
'edit_terms'    => 'edit_projects',
'delete_terms'  => 'edit_projects',
'assign_terms'  => 'edit_projects',
```
from the taxonomies fixes the issue, but why?

Steps to test this PR:

1. Create a new user using the LTW Editor role
2. Attempt to add, update, delete LTW projects and taxonomies
3. Make sure everything works as expected
4. Try all of the above as an administrator and editor as well
